### PR TITLE
Add android braze epic monitoring

### PR DESCRIPTION
**Pull request by cc-rock**
Additional:
- Lowered Olgil epic threshold for Android
- Added .DS_Store files to gitignore

### CODE REVIEW

File: src/main/scala/com/gu/datalakealerts/Features.scala [hunk 1]
```diff
@@ -82,7 +82,7 @@ object Features {
             |and ab.name like '%epic%'
             |and ab.completed = True
             |group by 1
-          """.stripMargin, 285000)
+          """.stripMargin, 42680)
```
Comment (cc-rock) [inline src/main/scala/com/gu/datalakealerts/Features.scala]: Olgil threshold set as half of yesterday's total impressions, as discussed with @jordanterry

File: src/main/scala/com/gu/datalakealerts/Features.scala [hunk 2]
```diff
@@ -111,6 +111,17 @@ object Features {
...
+          MonitoringQuery(s"""
+                             |select browser_version, count (distinct page_view_id)
+                             |from clean.pageview
+                             |cross join unnest (component_events) x (c)
+                             |where received_date = date '$yesterday'
+                             |and device_type like '%ANDROID%'
+                             |and c.component.type = 'APP_EPIC'
+                             |and c.action = 'VIEW'
+                             |group by 1
+          """.stripMargin, 124090)
```
Comment (cc-rock) [inline src/main/scala/com/gu/datalakealerts/Features.scala]: Braze threshold set as half of yesterday's total impressions, as discussed with @jordanterry

**Comment by cc-rock** [comment]
@alexanderedge @ddramowicz @alexduf 
Would you mind having a look when you have some time? Thank you!

File: cfn.yaml [hunk 3]
```diff
@@ -186,6 +186,13 @@ Resources:
               "featureId": "braze_epic",
               "platformId": "ios"
             }
+        - Arn: !GetAtt Lambda.Arn
+            Id: android-braze-epic-scheduler
```
Comment (alexanderedge) [inline cfn.yaml]: The indentation of `Id` and `Input` do not match the ones above. My knowledge of yaml is limited but I would expect this to not work correctly with ClourFormation. Have you deployed this to CODE environment using RiffRaff?

Comment (cc-rock) [reply cfn.yaml]: Thanks.. infact it didn't work. Fixed now.

File: README.md [hunk 4]
```diff
@@ -27,10 +27,10 @@ a different table, you'll need to add further permissions to the Cloudformation
 
 ### Testing your changes
 
-Ensure you are on the correct AWS region (`eu-west-1`). This can be achieved by using the `AWS_DEFAULT_REGION` environment variable:
+Ensure you are on the correct AWS region (`eu-west-1`). This can be achieved by using the `AWS_REGION` environment variable:
```
Comment (cc-rock) [inline README.md]: @alexduf AWS_DEFAULT_REGION was not working for me, after googling I found that the variable name is AWS_REGION, which infact worked.

**Pull request merged**